lib/resources/r4/encounter.yaml

```diff
@@ -601,9 +627,9 @@ fields:
 
   note: >
     <ul>
-      <li>custom-attribute-name and custom-attribute-value are both required</li>
-      <li>id and valueString for custom-attribute-name are required</li>
-      <li>custom-attribute-value can be of type string or CodeableConcept</li>
+      <li>custom-attribute-name and custom-attribute-value are both required.</li>
+      <li>id and valueString for custom-attribute-name are required.</li>
+      <li>custom-attribute-value can be of type String, Integer, DateTime, or CodeableConcept.</li>
```

Comment:
string, integer, and dateTime are lower case bc they're primitive types

Reply:
updated https://github.com/cerner/fhir.cerner.com/pull/825/commits/1f51d6f592da1b972737f18b7ab944114ebfe8b9